docs/README.md

```diff
@@ -14,6 +14,84 @@ limitations under the License.
 
 # Documentation
 
-No documentation available yet. Start your first contribution with some documentation.
+> Highly customizable open source theme for Hugo based static websites
+
+Syna is a Hugo template built to be customizable and easy to use at the same time. To achieve this we have built the theme in a way that all the pages would be created by using building blocks
+called fragments. Each page of a website using Syna is created using fragments.
+
+## Fragments
+
+Fragments are the way you define your view. There are several prebuilt fragments that are necessary to a website and users can write their own fragments and even override the default fragments. These fragments are controlled with page resources or fragments defined in a headless page called `global` which would be rendered in all pages.
+
+Fragments themselves are partials that are located in `layouts/partials/fragments`. When a page is being rendered and is using the default layout `single` (no need to be explicitly mentioned in the page bundle), Syna would do the following:
```

Comment:
This is developer documentation and should most likely be separated off and linked to. Something like: docs/dev/README.md or so.